database/seeders/PersonalAccessTokenSeeder.php

```
@@ -0,0 +1,115 @@
+     */
+    public function run(): void
+    {
+        // Only run in development environment
+        if (app()->environment('production')) {
+            $this->command->warn('Skipping PersonalAccessTokenSeeder in production environment');
+
+            return;
+        }
+
+        // Get the first user (usually the admin user created during setup)
+        $user = User::find(0);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\User' in method 'run'. (undefined)

(StaticAccess)